packages/components/AddCommentButton/AddCommentButton.tsx

```diff
@@ -57,7 +57,7 @@ export const AddCommentButton = forwardRef<HTMLButtonElement, AddCommentButtonPr
         onMouseLeave={clearHover}
         className={buttonClassnames}
       >
-        <Icon name="comment-plus" fill="currentColor" />
+        <Icon name="comment-plus" className="p-0.5" />
```

Comment:
This class actually _has_ a CSS module file (`AddCommentButton.module.css`) Can we add this style there rather than using a global Tailwind selector? I know we have them mixed in the codebase but IMO it's nice to at least avoid mixing the two _within a component_.

Reply:
Yeah, for sure! I'll get it updated. I need to come back through here and move the rest of these components over to the design package in a follow-up.

Reply:
Mind just adding it before this lands though? 😅

Reply:
Yeah, for sure! I did 😊 just meant that I need to get everything moved over still.

Reply:
Oh lol sorry